uber/config.py

```diff
@@ -303,6 +303,28 @@ def REMAINING_BADGES(self):
     def ADMIN_ACCESS_SET(self):
         return sa.AdminAccount.access_set()
 
+    @request_cached_property
+    def EMAIL_APPROVED_IDENTS(self):
+        return self.get_approved_idents()
+
+    # TODO: only reason this is a separate method is because for unit tests we patch this method,
+    # and monkeypatch.setattr() doesn't like to patch EMAIL_APPROVED_IDENTS
+    # when it's decorated with @request_cached_property
+    def get_approved_idents(self):
+        with sa.Session() as session:
+            return {ae.ident for ae in session.query(sa.ApprovedEmail)}
+
```

Comment:
In general when monkeypatching properties you'll typically need to patch the class and not the instance.  When using something like ``@property`` or ``@cached_property`` or ``@request_cached_property`` then the property is readonly, so you'll get an exception.

Though with that being said, "no attribute found" is a different exception than I would have expected to see!  But I would expect that if you said
```
from uber.config import Config
...
monkeypatch.setattr(Config, 'EMAIL_APPROVED_IDENTS', ['mock_ident'])
```
then I'd expect that to work.